rpc/trace_test.go

```diff
@@ -34,3 +38,49 @@ func TestNotImplemented(t *testing.T) {
 		}
 	}
 }
+
+// TestSimulateTransaction tests the SimulateTransaction method
+func TestSimulateTransaction(t *testing.T) {
+	testConfig := beforeEach(t)
+
+	var simulateTxIn SimulateTransactionInput
+	var expectedResp SimulateTransactionOutput
+	if testEnv == "mainnet" {
+		simulateTxnRaw, err := os.ReadFile("./tests/simulateInvokeTx.json")
+		require.Nil(t, err, "Error ReadFile simulateInvokeTx")
+
+		err = json.Unmarshal(simulateTxnRaw, &simulateTxIn)
+		require.Nil(t, err, "Error unmarshalling simulateInvokeTx")
```

Comment:
Personally I find these comments help me quickly identify what went wrong and where the error arose. I can remove them if you it would be better to do so